lib/gems/pending/util/miq-process.rb

```diff
@@ -111,7 +111,7 @@ def self.processInfo(pid = nil)
       percent_cpu                    = (1.0 * result[:cpu_time]) / cpu_total
       result[:percent_cpu]           = round_to(percent_cpu * 100.0, 2)
 
-      smaps = Sys::ProcTable.ps(pid).smaps
+      smaps = Sys::ProcTable.ps(:pid => pid).smaps
```

Comment:
@djberg96 So I was thinking about this when you made this PR, but what are your thoughts about replacing all of the functionality found here (in `MiqProcess`) with current (or new) functionality in `Sys::ProcTable`?

There might be a few changes we want to consider to do that, since we probably don't need to load *everything* that `.ps` loads for each of the methods in this file, but I think it just makes sense as another thing we can remove from this repo.

Reply:
Looking at `MiqProcess`, it seems *most* of it could be replaced, but I would have to do some more research.

Reply:
Yeah, I guess it isn't a perfect 1-to-1 port, but it would be nice to remove a lot of it if nothing else.